packages/core/src/types/EventTypes.ts

```diff
@@ -225,8 +226,8 @@ type ImageSpacingCalibratedEventDetail = {
   viewportId: string;
   renderingEngineId: string;
   imageId: string;
-  rowScale: number;
-  columnScale: number;
+  scale?: number;
+  calibration?: IImageCalibration;
```

Comment:
does `scale` here live because of legacy implementation? since there is scale inside calibration too.

Reply:
I'm removing scale, and adding some more documentation.
Also adding an aspect ratio, but the code currently does not handle it effectively.  I've figured out how to fix that, but I'm not quite sure yet where the changes would be.  The correct way to handle aspect ratio is to:
1. Render the image in the specified aspect ratio
2. Add an SVG transform to apply the aspect ratio as well as pan/zoom settings